src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html

```diff
@@ -169,8 +59,7 @@
             </label>
             <textarea class="input__long"
                       id="request-information-location"
-                      required>
-            </textarea>
+                      required></textarea>
```

Comment:
I noticed there was a lot of tabs and blank spaces already entered when I did that.

So when I read this page on a screenreader it would say: `Request Information. Textfield. Text entered. SPACE SPACE SPACE SPACE SPACE SPACE SPACE...`